src/Symfony/Component/Mailer/Bridge/Mailgun/Tests/Webhook/Fixtures/clicks.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+use Symfony\Component\RemoteEvent\Event\Mailer\MailerEngagementEvent;
+
+$wh = new MailerEngagementEvent(MailerEngagementEvent::CLICK, 'Ase7i2zsRYeDXztHGENqRA', json_decode(file_get_contents(str_replace('.php', '.json', __FILE__)), true)['event-data']);
```

Comment:
Let's make `json_decode()` throw if someone messes up the fixtures.

```suggestion
$wh = new MailerEngagementEvent(MailerEngagementEvent::CLICK, 'Ase7i2zsRYeDXztHGENqRA', json_decode(file_get_contents(str_replace('.php', '.json', __FILE__)), true, flags: JSON_THROW_ON_ERROR)['event-data']);
```

Same for the other fixtures in this folder.